webpages/src/lib/serialize.js

```diff
@@ -2,6 +2,7 @@ const promisify =
   (callbackFn) =>
   (...args) =>
     new Promise((resolve) => callbackFn(...args, resolve));
+const browserLevelPermissions = ["notifications"];
```

Comment:
What about `"clipboardWrite"`?

Couldn't you just import this into App.vue?

Reply:
Yeah, this also needs the `if (typeof browser !== "undefined")` check for clipboardWrite.